backends-common/postgres/src/main/java/org/apache/james/backends/postgres/quota/PostgresQuotaCurrentValueDAO.java

```diff
@@ -69,7 +69,7 @@ public Mono<Void> decrease(QuotaCurrentValue.Key quotaKey, long amount) {
             .set(IDENTIFIER, quotaKey.getIdentifier())
             .set(COMPONENT, quotaKey.getQuotaComponent().getValue())
             .set(TYPE, quotaKey.getQuotaType().getValue())
-            .set(CURRENT_VALUE, 0L)
+            .set(CURRENT_VALUE, -amount)
```

Comment:
Because it makes the contract test fail
https://github.com/apache/james-project/blob/0b3e434f2bc48bbd7a517b2d9f046eafaef1f8a7/mailbox/store/src/test/java/org/apache/james/mailbox/store/quota/CurrentQuotaManagerContract.java#L86